### DISCUSSION TITLE
[组队] Web3D - 知识地图 label 可视化

### DISCUSSION THREAD

**Discussion by Demian101** [NonceGeek/Move-Camp#164] [Amazing Things!]
Web3 世界真是太精彩啦！
> 醒醒，你根本学不完。

可以拖拽的，最好能结合 VR 的 ，
> 可是，这和区块链又有什么关系呢。。。。

对前端的要求可能比较高

需求：
[前端] * 2 
[合约] * 1 
[吃瓜] * n

#组队
#不成熟 idea

**Comment by Demian101** [comment]
Orz 
怎么加组队 label...

**Comment by ShirleneLiu** [comment]
这又是为家属做的吧...
